src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int oneHundred = 100;

```

Comment:
```suggestion
    private static final int oneHundred = 100;
    private ColorSupplier = new ColorSupplier();
    private Random = new Random();
```
make these objects belong to class instance - they will be created only once for each instance, per your current implementation they are recreated each time the method is called